README.md

```diff
@@ -73,23 +73,59 @@ Then you can pull using:
 git pull-locales <branch>
 ```
 
-## manage
+## Submitting a translation job
+
+Once in a while the texts in this repository get sent to professional translators.
+The general process for the [./locales.json](locales.json) is as follows:
+
+* Create an issue that a new translation job should be created
+* Create a branch for the issue
+* Create the directory `./external-jobs/<identifier>/sent`, where `<identifier>` is `year>-<month>-<optional_key>`
+* Run `./manage convert ./locales.json ./external-jobs/<identifier>/sent`
```

Comment:
Does this create ods files or why did we send ods files? Are the translation offices incompatible with csv?

Reply:
AFAIK it's easier to say what should not be translated. We kind of didn't know what format they prefer and in the beginning the malteser said they want to translate it themselves and only can work with excel sheets. As linguarum wrote in the mail with the translated locales that they prefer to work with ods files, I guess we should keep it this way.

Reply:
To create ods files a separate tool is needed. I edited the readme accordingly.